.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: elixir
+elixir:
+  - 1.4.5
+  - 1.5.1
+otp_release:
+  - 19.3
+  - 20.0
+services:
+  - docker
+env:
+  - GOOGLE_APPLICATION_CREDENTIALS=$HOME/credentials.json
```

Comment:
Is the project set in the travis config?

Reply:
Yes! In the `GOOGLE_PROJECT_ID` environment variable. We *could* set this here as well, but I figure this is more straightforward.